# 不要打印下面这样的 warning，两个路径的区别只有盘符的大小写
# 更好的修复方法是在解析、创建 webpack 模块时就统一路径的格式
# WARNING in ./node_modules/.pnpm/css-loader@6.7.3_webpack@5.75.0/node_modules/css-loader/dist/runtime/api.js
# There are multiple modules with names that only differ in casing.
# This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
# Use equal casing. Compare these module identifiers:
# * D:\2\ddb\web\node_modules\.pnpm\source-map-loader@4.0.1_webpack@5.75.0\node_modules\source-map-loader\dist\cjs.js!D:\2\ddb\web\node_modules\.pnpm\css-loader@6.7.3_webpack@5.75.0\node_modules\css-loader\dist\runtime\api.js
#     Used by 3 module(s), i. e.
#     D:\2\ddb\web\node_modules\.pnpm\css-loader@6.7.3_webpack@5.75.0\node_modules\css-loader\dist\cjs.js??ruleSet[1].rules[2].use[1]!D:\2\ddb\web\node_modules\.pnpm\sass-loader@13.2.0_sass@1.58.0_webpack@5.75.0\node_modules\sass-loader\dist\cjs.js??ruleSet[1].rules[2].use[2]!D:\2\ddb\web\node_modules\.pnpm\xshell@1.0.7_react-dom@18.2.0\node_modules\xshell\scroll-bar.sass
# * D:\2\ddb\web\node_modules\.pnpm\source-map-loader@4.0.1_webpack@5.75.0\node_modules\source-map-loader\dist\cjs.js!d:\2\ddb\web\node_modules\.pnpm\css-loader@6.7.3_webpack@5.75.0\node_modules\css-loader\dist\runtime\api.js
#     Used by 11 module(s), i. e.
#     d:\2\ddb\web\node_modules\.pnpm\css-loader@6.7.3_webpack@5.75.0\node_modules\css-loader\dist\cjs.js??ruleSet[1].rules[2].use[1]!d:\2\ddb\web\node_modules\.pnpm\sass-loader@13.2.0_sass@1.58.0_webpack@5.75.0\node_modules\sass-loader\dist\cjs.js??ruleSet[1].rules[2].use[2]!d:\2\ddb\web\console\dashboard.sass
#  @ ./node_modules/.pnpm/css-loader@6.7.3_webpack@5.75.0/node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[2].use[1]!./node_modules/.pnpm/sass-loader@13.2.0_sass@1.58.0_webpack@5.75.0/node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[2].use[2]!./node_modules/.pnpm/xshell@1.0.7_react-dom@18.2.0/node_modules/xshell/scroll-bar.sass 3:0-127 4:30-57
#  @ ./node_modules/.pnpm/xshell@1.0.7_react-dom@18.2.0/node_modules/xshell/scroll-bar.sass 8:6-291 22:17-24 26:0-261 26:0-261 27:22-29 27:33-47 27:50-64
#  @ ./console/window.tsx 1:0-32

diff --git a/lib/WebpackOptionsApply.js b/lib/WebpackOptionsApply.js
index 96a485f7400a0bc49b1ab8b2b88f36f157ce7840..2651f3222084bbc901a9bf20ed1bd769a557f039 100644
--- a/lib/WebpackOptionsApply.js
+++ b/lib/WebpackOptionsApply.js
@@ -587,7 +587,7 @@ class WebpackOptionsApply extends OptionsApply {
 			portableIds: options.optimization.portableRecords
 		}).apply(compiler);
 
-		new WarnCaseSensitiveModulesPlugin().apply(compiler);
+		// new WarnCaseSensitiveModulesPlugin().apply(compiler);
 
 		const AddManagedPathsPlugin = require("./cache/AddManagedPathsPlugin");
 		new AddManagedPathsPlugin(